cmd/template.go

```diff
@@ -0,0 +1,56 @@
+/*----------------------------------------------------------------
+ *  Copyright (c) ThoughtWorks, Inc.
+ *  Licensed under the Apache License, Version 2.0
+ *  See LICENSE in the project root for license information.
+ *----------------------------------------------------------------*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/getgauge/gauge/logger"
+	"github.com/getgauge/gauge/template"
+	"github.com/spf13/cobra"
+)
+
+var (
+	templateCmd = &cobra.Command{
+		Use:     "template [flags] [args]",
+		Short:   "Change template configurations",
+		Long:    `Change template configurations.`,
+		Example: `  gauge template java getgauge/java-template`,
```

Comment:
`gauge init java`   fails with   `Failed to initialize project. Get getgauge/java-template: unsupported protocol scheme ""`  when template is configured using`gauge template java getgauge/java-template`

Reply:
@negiDharmendra As per discussion with @zabil  only absolute URLs are supported.
I will try to improve the error message.

Reply:
> As per discussion with @zabil only absolute URLs are supported.

Then we should change example uses to have absolute url.

Reply:
```suggestion
		Example: `  gauge template custom https://example.org/templates/custom-template.zip`,
```

Reply:
I think it is better to parse when the user is trying to add a template for example instead of 

```
gauge template java invalid/url
```

It should be 

```
gauge template java invalid/url

Error ----------------------------------

[Gauge]
'Failed to add template 'invalid/url'. The template location must be a valid (https) URI
```

The user can side step this by adding it directly to the config file. So display this error message when it fails before fetching too.